src/App.jsx

```diff
@@ -14,57 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const selectedGoodReset = () => setSelectedGood(null);
```

Comment:
```suggestion
  const handleSelectedGoodReset = () => setSelectedGood(null);
```
Function names should start from a verb, and in react developers have a convention on how to name handlers, https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364